### DISCUSSION TITLE
【疑问】关于使用三方播放后官方统计数据

### DISCUSSION THREAD

**Discussion by Doordoorjay** [qier222/YesPlayMusic#1368] [Q&A] [answered]
如果使用像此类第三方播放器，个人网易云账户内的“歌曲播放量”和“歌曲播放总数”会增加吗

**Comment by xhych** [comment]
> 看上去好像会 但是不会增加歌单播放量

**Comment by Doordoorjay** [comment]
> > 看上去好像会 但是不会增加歌单播放量

歌单播放啥的无所谓（）主要是想升级hhhh

**Comment by memorydream** [comment, marked as answer]
会记录，ypm在音乐库中有听歌排行，可以在网易云的相关页面中查看，数据都是一样的
